packages/sdk/src/protocols/store.ts

```diff
@@ -0,0 +1,317 @@
+import { sha256 } from "@noble/hashes/sha256";
+import { StoreCore, waku_store } from "@waku/core";
+import {
+  Cursor,
+  IDecodedMessage,
+  IDecoder,
+  IStoreSDK,
+  type Libp2p,
+  PageDirection,
+  type ProtocolCreateOptions
+} from "@waku/interfaces";
+import { ensurePubsubTopicIsConfigured, isDefined } from "@waku/utils";
+import { concat } from "@waku/utils/bytes";
+
+import { utf8ToBytes } from "../index.js";
+
+import { BaseProtocolSDK } from "./base_protocol.js";
+
+export const DefaultPageSize = 10;
+
+// const log = new Logger("sdk:store");
+
+const DEFAULT_NUM_PEERS = 1;
+
+export class StoreSDK extends BaseProtocolSDK implements IStoreSDK {
+  public readonly protocol: StoreCore;
+
+  constructor(libp2p: Libp2p, options?: ProtocolCreateOptions) {
+    // options.numPeersToUse is disregarded: https://github.com/waku-org/js-waku/issues/1685
```

Comment:
I think this can have `TODO` prefix

Reply:
8b6929efe000edd28ecd69940064adec7aade2bd